frontend/cli/cmd_serve.go

```diff
@@ -122,6 +134,13 @@ func (s *serveCmd) run(ctx context.Context, projConfig projectconfig.Config, ini
 		s.CommonConfig.AllowOrigins = append(s.CommonConfig.AllowOrigins, addr)
 	}
 
+	provisionerAddresses := make([]*url.URL, 0, s.Provisioners)
+	provisionerIngressAddresses := make([]*url.URL, 0, s.Provisioners)
+	for range s.Provisioners {
+		provisionerIngressAddresses = append(provisionerIngressAddresses, bindAllocator.Next())
```

Comment:
What is "ingress" in the context of the provisioner? For the Controller that's the separate address for HTTP ingress requests, but that doesn't make much sense for the provisioner...

Reply:
Ah, this is my misunderstanding. Will be removed Pronto!